### ISSUE TITLE
33 assemblies to use EF Core 2.0 on .NET Framework 4.7.1

### ISSUE THREAD

**Issue by MarcoLoetscher** [dotnet/efcore#10968]
Hello EF Core Team,
the installation of EF Core 2.0 on .NET Framework 4.7.1 requires 33 assemblies (Install-Package Microsoft.EntityFrameworkCore.SqlServer: 31 assemblies and Install-Package Microsoft.EntityFrameworkCore.Tools: 2 assemblies). Are there any plans to reduce this number?

**Comment by bricelam** [comment]
Note, this is primarily caused by the way .NET Standard 1.x works. If all our dependencies targeted 2.0, it would be A LOT less.

**Comment by MarcoLoetscher** [comment]
[EF Core now targets .NET Standard 2.0](https://docs.microsoft.com/en-us/ef/core/what-is-new/)? When will .NET Standard 1.x be cut off completely?

**Comment by ajcvickers** [comment]
@onovotny: Would you consider moving IX-Async to depend on only .NET Standard 2.0 and above, or do you still need to support 1.x dependencies?
@MichaelKetting: Same question for Relinq?

**Comment by onovotny** [comment]
@ajcvickers I'm curious as to the reason for the question. A library shouldn't need to target a higher .NET Standard version than the API set it requires, and IX-Async currently doesn't use any API's that are only in .NET Standard 2. So what does moving to be .NET Standard 2 gain?

**Comment by onovotny** [comment]
It also begs the question, is this a packages.config scenario or a PackageReference scenario? It is true that packages.config can add a lot of cruft, but the solution there should be to move to PackageReference where this all goes away.

**Comment by MichaelKetting** [comment]
@ajcvickers 
Would the switch to .NET Standard 2.0 be an additional dependency entry in the NuGet package or would this require the removal of .NET Standard 1.0? Right now, re-linq comes with dedicated targets for .NET 3.5, 4.0, 4.5, as well as .NET Portable net45+win+wpa81+wp80 and .NET Standard 1.0. What part of the support would I lose by replacing .NET Standard 1.0 with 2.0? I'm guessing mostly other libraries that only support .NET Standard 1.0.

Originally, I've integrated .NET Standard first and formost for EF Core, so deprecating .NET Standard 1.0 might be okay. It would definitely be something to announce/get feedback on, on the re-linq mailing list to make sure it doesn't hurt somebody.

**Comment by ajcvickers** [comment]
Thanks guys for your replies. We discussed this again today and came to the following conclusions:
* It's not a PackageReference verses packages.config issue; it's a .NET Core 1.1 issue that is a carry over from the PCL days
* 1.1 is certainly more and more emphasized, so at some point removing it as a target may be the right thing for our external dependencies to do, but we're not requesting or suggesting that now
* If 1.1 remains as a target, then fixing this will probably require IX-Async and Relinq to cross compile against 1.1 and 2.0. However, it's really up to the owners of those packages whether to do this, and the additional work, etc. may not be worth it to reduce the number of packages installed.

Closing this issue as there is nothing actionable on the EF side.
